Navbars/Side_Navbar_02/code/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Navbar Style</title>
+</head>
+<body>
+    <details class="menuBox">
```

Comment:
Avoid using custom html tags. Since we are making components for developers to use, some might not be familiar with them and can lead to confusion

Reply:
Please review now!